packages/components/organisms/f-checkout/package.json

```diff
@@ -61,7 +61,6 @@
   "devDependencies": {
     "@babel/plugin-proposal-class-properties": "7.12.13",
     "@justeat/f-alert": "0.5.0",
-    "@justeat/f-button": "1.2.0",
```

Comment:
you've removed the `f-button` completely?

Reply:
Haha, I think I removed it to test why the yarn.lock wasn't adding the new version. Back now